src/Config/Services.php

```diff
@@ -30,8 +30,7 @@ public static function settings(?SettingsConfig $config = null, bool $getShared
             return static::getSharedInstance('settings', $config);
         }
 
-        /** @var SettingsConfig $config */
-        $config = $config ?? config('Settings');
+        $config ??= config('Settings');
 
         return new Settings($config);
```

Comment:
The only reason this was separate was for the inline type. If that isn't needed we don't need the intermediate variable. 
```php
return new Settings($config ?? config('Settings'));
```

Reply:
Done!